examples/demo-vendoring/atmos.yaml

```diff
@@ -16,6 +16,16 @@ stacks:
     - "**/_defaults.yaml"
   name_pattern: "{stage}"
 
+vendor:  
+  # Single file
+  base_path: "./vendor.yaml"
+  
+  # Directory with multiple files
+  #base_path: "./vendor"
+  
+  # Absolute path
+  # base_path: "nested-vendor/vendor1.yaml"
```

Comment:
I don't like to give things a name like this, because inevitably someone will copy this.

Let's call the folder `vendor.d` in our example

Reply:
following in line with the `.d` convention (e.g. `init.d` and `conf.d`)

Reply:
resolved